compose/ui/ui/src/webTest/kotlin/androidx/compose/ui/OnCanvasTests.kt

```diff
@@ -43,14 +44,20 @@ internal interface OnCanvasTests {
         return canvas
     }
 
-    fun commonAfterTest() {
+    private fun commonAfterTest() {
```

Comment:
this method seems to be redundant? Can be merged with resetCanvas?

Reply:
@eymar the only purpose of this method as a separate entity was to sort of remind that we can be back to `@Before` / `@After` test setup if needed - after moving to Kotlin 2.0 - but I agree that now it's even confusing - so let's may be just merge them indeed